plenum/common/ledger_manager.py

```diff
@@ -86,6 +88,29 @@ def addLedger(self, iD: int, ledger: Ledger,
             verifier=MerkleVerifier(ledger.hasher)
         )
 
+    def _cancel_request_ledger_statuses_and_consistency_proofs(self, ledger_id):
+        if ledger_id in self.request_ledger_status_action_ids:
+            self._cancel(self.request_ledger_status_action_ids[ledger_id])
+            self.request_ledger_status_action_ids[ledger_id] = None
+
+        if ledger_id in self.consistency_proof_action_ids:
+            self._cancel(self.consistency_proof_action_ids[ledger_id])
+            self.consistency_proof_action_ids[ledger_id] = None
+
+    def request_ledger_status_if_needed(self, ledger_id):
+        ledgerInfo = self.getLedgerInfoByType(ledger_id)
+        nodes = [node for node in self.owner.nodeReg if node not in ledgerInfo.ledgerStatusOk]
+        self.owner.request_ledger_status_from_nodes(ledger_id, nodes)
+
+    def request_last_CP(self, ledgerId):
```

Comment:
It would be better to name this method `reask_for_last_CP`.

Reply:
Done.